cmd/zfs/zfs_main.c

```diff
@@ -4504,6 +4504,8 @@ zfs_do_send(int argc, char **argv)
 				    gettext("missing argument for '%s' "
 				    "option\n"), argv[optind - 1]);
 			}
+			if (excludes.list != NULL)
+				free(excludes.list);
```

Comment:
We can at least live without these `!= NULL` checks; `free(3)` will handle a NULL as a no-op on all currently supported platforms.

Reply:
I am always surprised when I hear this and this time, I checked:

https://pubs.opengroup.org/onlinepubs/9699919799.2008edition/
https://www.man7.org/linux/man-pages/man3/free.3.html

You are right that the specification says this. The coverity model might need to be updated to take this into account.

That said, I will repush with the NULL checks removed.

Reply:
Repushed.